ansible/requirements.yml

```diff
@@ -1,4 +1,5 @@
 collections:
+  - ansible.posix
   - community.general
```

Comment:
If you know what modules you are using from `ansible.posix` and `community.general` then you might think about adding a `version` key here [à la cisagov/ansible-role-vnc-server](https://github.com/cisagov/ansible-role-vnc-server/blob/207d8d52588bbc59f2dabfb1e021bb00344ec80d/meta/requirements.yml#L12-L17)

Reply:
When I looked into this (I had the same thought) the module from `ansible.posix` used was included in the 1.0 release with the options we are using. As a result I decided not to worry about adding a `version` key. It appears that the module used from `community.general` with the options we are using has been present since before it was broken out of being included with Ansible.